### PR TITLE
Fix caution visual bug for dark theme 

### PR DESCRIPTION
**Pull request by Notenlish**
Old:
image
New:
image

reported by @yunline

### CODE REVIEW

**Comment by Notenlish** [comment]
https://github.com/pygame-community/pygame-ce/issues/2417

**Review by yunline — CHANGES_REQUESTED** [review]
Thanks for your fix.
However, there are 2 bugs mentioned in the issue, and you just fixed one of them.
The another one is in https://pyga.me/docs/tutorials/en/move-it.html
image
image

**Comment by Starbuck5** [comment]
@yunline It's fine for this PR to only fix one of the problems, as long as the original issue doesn't get closed.

File: docs/reST/themes/classic/theme.conf [hunk 1]
```diff
@@ -53,6 +53,7 @@ dark_h1color                = #e1cd62
 dark_h2color                = #00b800
 dark_notebgcolor            = #555349
 dark_notebordercolor        = #6e6b5e
+dark_cautionbgcolor         = rgb(255 25 25 / 30%)
```
Comment (Starbuck5) [inline docs/reST/themes/classic/theme.conf:56]: Opacity in the css? This is single-handedly bringing pygame-ce into the future :) /s

Comment (novialriptide) [reply docs/reST/themes/classic/theme.conf:56]: Should be a hex value. `/ 30%` shouldn't be here as well.

Comment (Starbuck5) [reply docs/reST/themes/classic/theme.conf:56]: I think it's fine for it to have opacity. It looks good in the output.

**Comment by rethanon** [comment]
Just wanted to point out that we already have a color we use for a warning in the _sdl2 docs, should we try to be consistent and use the same color?
image

**Comment by rethanon** [comment]
Which I guess does lead to the question, rather than just apply changes to the dark mode, could we apply the background color/text color universally like we have for the above warning in sdl2_video?

File: docs/reST/themes/classic/static/pygame.css_t [hunk 2]
```diff
@@ -735,6 +735,23 @@ div.caution {
...
+    color: #BA1200;
+    font-size: 1.2rem;
+}
+.dark-theme div.admonition.caution {
+    padding: 10px;
+}
+.dark-theme div.admonition.caution code {
+    background-color: transparent;
+}
+
```
Comment (MyreMylar) [inline docs/reST/themes/classic/static/pygame.css_t:754]:
```suggestion
.dark-theme div.caution {
    background-color: {{ theme_dark_cautionbgcolor }};
    border: 1px solid;
    border-color: {{ theme_dark_cautionbordercolor }};
}

```

File: docs/reST/themes/classic/theme.conf (hunk 1 above)
Comment (MyreMylar) [inline docs/reST/themes/classic/theme.conf:56]:
```suggestion
dark_cautionbgcolor         = rgb(255 25 25 / 30%)
dark_cautionbordercolor     = rgb(255 60 60)
```

**Comment by MyreMylar** [comment]
I think it is fine for us to have different colours for Warning and Caution boxes.

I applied the suggestions from the earlier comments so we can get this merged.

Caution boxes in the dark theme now look like this:

image

**Review by MyreMylar — APPROVED** [review]
LGTM now.

**Pull request merged**